prompts/configs.go

```diff
@@ -308,19 +310,26 @@ func getValuesForField(
 	if field.Description != nil {
 		description = *field.Description
 	}
-	if field.Name == "packageName" && isQuickstart && sdkClassName != "" {
+	if field.Name == "packageName" && isQuickstart {
```

Comment:
What is the significance of omitting sdkClassName from this `if`?

Reply:
Means that it will be shown by default which I think is ok